Interrogator.xUnit/NotConcurrentAttribute.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Interrogator.xUnit.Common;
+using Interrogator.xUnit.Utilities;
+
+namespace Interrogator.xUnit
+{
+	[AttributeUsage(AttributeTargets.Method, AllowMultiple = true)]
+	public class NotConcurrentAttribute : DependsOnAttribute
+	{
+		public enum ConcurrencyScope
+		{
+			Class,
+			Namespace,
+			Assembly
+		}
+
+		public string GroupName { get; }
+		public ConcurrencyScope Scope { get; }
+		public string CurrentMethodName { get; }
+
+		public NotConcurrentAttribute(string groupName, [CallerMemberName]string currentMethodName = null) : this(groupName, ConcurrencyScope.Class, currentMethodName) { }
```

Comment:
Can you make overloads where groupname is optional please? Then just using NotConcurrent without any parameters means NotConcurrent in this class only.

Reply:
This would let you just do ```NotConcurrent``` in the common case.